livekit-plugins/livekit-plugins-elevenlabs/livekit/plugins/elevenlabs/tts.py

```diff
@@ -485,7 +485,6 @@ async def recv_task():
                 if data.get("audio"):
```

Comment:
lgtm, I think it's also because: 
```suggestion
                if "audio" in data:
```